docs/pip/compatibility.md

```diff
@@ -148,10 +148,9 @@ supports the following values:
 While `unsafe-best-match` is the closest to `pip`'s behavior, it exposes users to the risk of
 "dependency confusion" attacks.
 
-In the future, uv will support pinning packages to dedicated indexes (see:
-[#171](https://github.com/astral-sh/uv/issues/171)). Additionally,
-[PEP 708](https://peps.python.org/pep-0708/) is a provisional standard that aims to address the
-"dependency confusion" issue across package registries and installers.
+uv also supports pinning packages to dedicated indexes (see:
```

Comment:
Should we refer to "searching across multiple indexes" and deduplicate the content above?

Reply:
I could go either way. I don't mind having dedicated content here on how it differs from pip.